java/code/src/com/suse/manager/webui/controllers/login/LoginController.java

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright (c) 2018 SUSE LLC
```

Comment:
If this is a new class you want to put 2019 here.

Reply:
Fixed